exonum/src/events/noise/wrappers/sodium_wrapper/wrapper.rs

```diff
@@ -78,10 +78,9 @@ impl NoiseWrapper {
         Ok(buf)
     }
 
-    pub fn write_handshake_msg(&mut self) -> Result<Vec<u8>, NoiseError> {
-        // Payload in handshake messages can be empty.
+    pub fn write_handshake_msg(&mut self, msg: &[u8]) -> Result<Vec<u8>, NoiseError> {
         let mut buf = vec![0_u8; MAX_MESSAGE_LENGTH];
-        let len = self.write(&[], &mut buf)?;
+        let len = self.write(msg, &mut buf)?;
```

Comment:
I've reviewed the call stack and it seems that `noise` crate verifies message length and returns an error if `msg` is too big.

This is not a request for change, just a review remark in case anybody else is wondering whether we'll panic if it turns out that `msg.len() > buf.len()`, and whether we need a length check like in `read_handshake_msg()` in order to avoid a panic. It seems that we don't need them.